Sources/JavaScriptEventLoop/JavaScriptEventLoop.swift

```diff
@@ -116,6 +116,24 @@ public extension JSPromise {
             }
         }
     }
+
+    /// Wait for the promise to complete, returning its result or exception as a Result.
+    var result: Result<JSValue, JSValue> {
```

Comment:
Nice!
Don't we want this to be public?

Reply:
The extension is public, which makes everything declared within it public.